pallets/gear/src/manager.rs

```diff
@@ -405,4 +437,29 @@ where
             common::set_program_persistent_pages(program_id, persistent_pages);
         };
     }
+
+    fn send_value(&mut self, from: ProgramId, to: Option<ProgramId>, value: u128) {
+        let from = from.into_origin();
+        if let Some(to) = to {
+            let to = to.into_origin();
+            log::debug!(
+                "Value send of amount {:?} from {:?} to {:?}",
+                value,
+                from,
+                to
+            );
+            let from = <T::AccountId as Origin>::from_origin(from);
+            let to = <T::AccountId as Origin>::from_origin(to);
+            let _ = T::Currency::repatriate_reserved(
+                &from,
+                &to,
+                value.unique_saturated_into(),
+                BalanceStatus::Free,
+            );
+            return;
```

Comment:
I would place here `else` block also for visibility

Reply:
agreed